packages/axes/src/reactive.ts

```diff
@@ -28,7 +28,7 @@ export const REACTIVE_AXES: ReactiveAdapter<
   state: {},
```

Comment:
you added to create

The state can be omitted, and it can also be omitted from the framework.